### PR TITLE
Abhishek 1

### PR DESCRIPTION
**Pull request by abhisheksharan**
Merged master with branch, resolved conflicts and included my original code changes. code tested and working with master

### CODE REVIEW

File: src/BorrowBookUI.java [hunk 1]
```diff
@@ -85,12 +85,12 @@ public void run() {
 					
 				
 			case FINALISING:
-				String Ans = input("Commit loans? (Y/N): ");
+				String Ans = input("Commit loans? (Y/N): "); //was Ans
```
Comment (9Mcgusto6) [inline src/BorrowBookUI.java]: I'm sure "Ans" is a variable, so it should be changed to "ans".

File: src/book.java [hunk 2]
```diff
@@ -4,62 +4,62 @@
...
-	private int ID;
+	private String title; // was TITLE
+	private String author; // was AUTHOR
+	private String callNo; // was CALLNO
+	private int id; // was ID
 	
-	private enum STATE { AVAILABLE, ON_LOAN, DAMAGED, RESERVED };
-	private STATE State;
+	private enum State {AVAILABLE, ON_LOAN, DAMAGED, RESERVED}; //delete whitespace, STATE
+	private State State;
```
Comment (9Mcgusto6) [inline src/book.java]: "private State State" should be changed to "private State state".

**Review by 9Mcgusto6 — CHANGES_REQUESTED** [review]
I found two variables that I would change. Otherwise, it looks as though it is all done. Good job.

File: src/BorrowBookUI.java (hunk 1 above)
Comment (abhisheksharan) [reply src/BorrowBookUI.java]: great! has been now updated

File: src/book.java (hunk 2 above)
Comment (abhisheksharan) [reply src/book.java]: great! has been now updated

**Review by 9Mcgusto6 — APPROVED** [review]
As I can see it, everything is all in order. I approve of the merge.

**Review by RozalindB — APPROVED** [review]

**Pull request merged**